filebeat/registrar/registrar.go

```diff
@@ -188,7 +190,9 @@ func (r *Registrar) Run() {
 
 		beforeCount := r.states.Count()
 		r.states.Cleanup()
```

Comment:
I initially was thinking about this but were undecided if states.Cleanup() should return the diff or the number of states that exist after the Cleanup. Will change it to return the diff.